src/__screenshot_tests__/tabs-screenshot-test.tsx

```diff
@@ -55,23 +55,25 @@ test('Tabs with long text and icon', async () => {
 });
 
 test('Tabs selected line appears properly', async () => {
-    await openStoryPage({
+    const page = await openStoryPage({
         id: 'components-tabs--default',
         device: 'MOBILE_IOS',
     });
 
+    const tabs = await screen.findByRole('tablist');
```

Comment:
Looks like storybook 7 is a bit slower than storybook 6 when loading stories. If we don't wait until the tabs are added to the DOM, `page.evaluate()` doesn't find the tab elements